web/themes/custom/server_theme/src/scss/components/typography.scss

```
@@ -1,3 +1,11 @@
 h1, h2, h3, h4 {
   @apply font-headers leading-none text-gray-700;
 }
+
+.fluid-container-wide {
+  @apply max-w-7xl mx-auto px-4 sm:px-6 lg:px-8;
```

```suggestion
  @apply max-w-7xl mx-auto px-4;
  
  @screen sm {
    @apply px-6;
  }
  
  @screen lg {
    @apply px-8;
  }
```

Do the responsive classes work with @apply? From what I can remember with Tailwind 1 it doesn't work. Maybe it works now?

---

Yeah, it's works with TW2

---

Neat!